src/ui/MessageItemReactionMenu/index.tsx

```diff
@@ -70,39 +83,39 @@ export function MessageEmojiMenu({
               closeDropdown={closeDropdown}
               spaceFromTrigger={spaceFromTrigger}
             >
-              {emojiContainer && getEmojiListAll(emojiContainer).map((emoji: Emoji): ReactElement => {
+              {filteredEmojis.map((emoji: Emoji): ReactElement => {
                 const isReacted: boolean = message?.reactions
                   ?.find((reaction: Reaction) => reaction.key === emoji.key)
                   ?.userIds
                   ?.some((reactorId: string) => reactorId === userId) || false;
                 return (
-                  <ReactionButton
-                    key={emoji.key}
-                    width="36px"
-                    height="36px"
-                    selected={isReacted}
-                    onClick={() => {
-                      closeDropdown();
-                      toggleReaction?.(message, emoji.key, isReacted);
-                    }}
-                    testID={`ui_emoji_reactions_menu_${emoji.key}`}
-                  >
-                    <ImageRenderer
-                      url={emoji.url}
-                      width="28px"
-                      height="28px"
-                      placeHolder={({ style }) => (
-                        <div style={style}>
-                          <Icon
-                            type={IconTypes.QUESTION}
-                            fillColor={IconColors.ON_BACKGROUND_3}
-                            width="28px"
-                            height="28px"
-                          />
-                        </div>
-                      )}
-                    />
-                  </ReactionButton>
+                    <ReactionButton
```

Comment:
The indentations here doesn't seem to be necessary. You can put this changes back too.

Reply:
Applied